src/lexedata/importer/fromexcel.py

```diff
@@ -140,7 +140,7 @@ def find_db_candidates(
         object: Ob,
         properties_for_match: t.Iterable[str],
         edit_dist_threshold: t.Optional[int] = None,
-    ) -> t.Iterable[str]:
+    ) -> t.List[str]:
```

Comment:
Why do we want to guarantee a list? All we need is that `iter` can be called on it.

Reply:
I think, I changed it, because this function actually returns a list. So why say it returns an Iterable if this specific function returns a list? Of course, I don't mind. Just my thoughts.

Reply:
The type annotations are the things we want to guarantee. And in this case, we do want to guarantee that the recipient of the function output can be iterated over. That it can be indexed and modified (the additional guarantees of a list) is stronger than the thing we actually need to promise, so we can be more conservative in our promises.